### PR TITLE
Keyword search for jobs

### PR DESCRIPTION
**Pull request by abhinaykrupa**

### CODE REVIEW

File: application/controllers/Jobs.php [hunk 1]
```diff
@@ -3,15 +3,7 @@
...
-    var $selectedSearchTypeId;
-    var $selectedMakeId;
-    var $selectedModelId;
-    var $selectedZip;
-    var $selectedDistanceId;
-    var $selectedPriceMin;
-    var $selectedPriceMax;
-    var $selectedYearMin;
-    var $selectedYearMax;
+    var $selectedKeyword;
```
Comment (abhinaykrupa) [inline application/controllers/Jobs.php]: @NSValli ...this is good but try to retain some fields that you might use in the future .. like $selectedSearchTypeId, $selectedZip, $selectedDistanceId ... you might use these fields in the job search going forward

Comment (abhinaykrupa) [inline application/controllers/Jobs.php]: @NSValli ... not just here but in all the places below where ever we use these variables

File: application/models/JobsModel.php [hunk 2]
```diff
@@ -204,11 +139,7 @@ class jobsModel extends CI_Model
...
         ],
-        'indeed' => [
-            'new' => 'New+Cars',
-            'used' => 'Used+Cars',
-            'certified' => 'Certified+Cars',
-            'all' => '',
+        'indeed' => [           
         ],
         'job' => [
             'new' => 'new',
```
Comment (abhinaykrupa) [inline application/models/JobsModel.php]: @NSValli do we need the below code variables ? or can we clean them up?

File: application/models/JobsModel.php [hunk 3]
```diff
@@ -293,184 +215,42 @@ public function getDistances()
     {
         return $this->distances;
     }
-
-    public function getLocation($zip = false)
-    {
-        $this->location = get_location($zip ? $zip : $this->zip);
-        return $this->location;
-    }
```
Comment (abhinaykrupa) [inline application/models/JobsModel.php]: @NSValli ... you might need the above method once you start working on the location of a specific job search

File: application/models/JobsModel.php [hunk 4]
```diff
@@ -293,184 +215,42 @@ public function getDistances()
...
      */
     public function getWebsites($params)
-    {
+    {       
         $websites = [];
-
-        if ($params['zip']) {
-            $this->zip = $params['zip'];
-            $this->getLocation();
-        }
```
Comment (abhinaykrupa) [inline application/models/JobsModel.php]: @NSValli you might need this code once you start working on the location search of a given job keyword soon

File: application/views/jobsView.php [hunk 5]
```diff
@@ -0,0 +1,48 @@
+<form id="search" name="<?= $pageId ?>" class="form-horizontal" action="<?= $canonicalURL ?>" method="GET">
+     <div class="form-group">
+        <label class="control-label col-sm-5" for="zip">Keyword:</label>
```
Comment (abhinaykrupa) [inline application/views/jobsView.php]: @NSValli change the label to 'Job Title'

**Comment by abhinaykrupa** [comment]
@NSValli review the comments

File: application/models/JobsModel.php (hunk 2 above)
Comment (kadarikalyani) [inline application/models/JobsModel.php]: yeah we will clean up them.

File: application/models/JobsModel.php (hunk 3 above)
Comment (kadarikalyani) [inline application/models/JobsModel.php]: ok

**Pull request merged**